pandas/core/resample.py

```diff
@@ -909,6 +906,8 @@ def interpolate(
         ``07:00:00`` and ``07:00:02``.
         """
         assert downcast is lib.no_default  # just checking coverage
+        if kwargs.get("inplace"):
```

Comment:
Could we disallow `inplace` without having a check here?

Reply:
Because it is passed twice you mean, I see. But this will not produce a useful error message for the user, will it? I still don't know what you have in mind, would be awesome if you could propose a precise improvement 🙏